build/tutorials-contest/liquid-avax/README.md

```diff
@@ -0,0 +1,7 @@
+This is our subscription to Avalanche developers contest. We have chosen the "How to create staking derivatives on Avalanche" subject.
+
+#Liquid-Avax
+
+Liquid staking is an alternative to normal staking. The advantage of liquid staking is that staked asset are not locked anymore, while still earning returns and supporting the network decentralisation.
```

Comment:
What is the purpose of the empty file `build/tutorials-contest/liquid-avax/Smart contracts/sAVAX.sol` ? Can it be removed?

Reply:
That is the ERC-20 token contract of the staked representation of 1 $AVAX. I totally forgot to include the content, it's done now